docs/src/content/docs/ko/guides/pages.mdx

```diff
@@ -109,26 +109,25 @@ Markdown 페이지의 프론트매터와 유사하게 이 페이지에 대한 [
 
 ##### `sidebar`
 
-**타입:** `SidebarEntry[]`
+**타입:** [`SidebarItem[]`](/ko/reference/configuration/#sidebaritem)
```

Comment:
```suggestion
**타입:** [`SidebarItem[]`](/ko/reference/configuration/#sidebaritem)  
```

This is missing the 2 trailing spaces so that `기본값` is on the next line.

Unrelated to this PR, but it also looks like these 2 spaces are missing for the `hasSidebar`, `headings`, `dir`, `lang`, and `isFallback` props following this part. Would you be up for fixing them at the same time?

Reply:
Thank you! I fixed it.